pmd-core/src/main/java/net/sourceforge/pmd/autofix/rule/DeleteRuleViolationFix.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.autofix.rule;
```

Comment:
not sure why this is in a "rule" child package from the interface...

Reply:
I deleted the DeleteRuleViolationFix class for the time being. However, I'm not yet sure where rule violation fixes which apply to any language can be placed.